<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="Nabila-PC" name="FrontendTesting.PageControllerTesting" tests="5" failures="1" timestamp="19 May 2015 10:21:20 GMT" time="0.170" errors="3">
  <testcase name="CreateMyTimelinePost" time="0.084" classname="FrontendTesting.PageControllerTesting">
=======
<<<<<<< HEAD
<testsuite hostname="Nabila-PC" name="FrontendTesting.PageControllerTesting" tests="4" failures="0" timestamp="15 May 2015 10:07:10 GMT" time="0.144" errors="4">
  <testcase name="CreateMyTimelinePost" time="0.057" classname="FrontendTesting.PageControllerTesting">
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
    <error type="java.lang.AssertionError" message="Failed expected:&lt;n&gt; but was:&lt;OK&gt;">
      <![CDATA[java.lang.AssertionError: Failed expected:<n> but was:<OK>
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
<<<<<<< HEAD
	at FrontendTesting.PageControllerTesting.CreateMyTimelinePost(PageControllerTesting.java:154)
=======
	at FrontendTesting.PageControllerTesting.CreateMyTimelinePost(PageControllerTesting.java:143)
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- CreateMyTimelinePost -->
<<<<<<< HEAD
  <testcase name="CreatePage" time="0.021" classname="FrontendTesting.PageControllerTesting">
    <error type="java.lang.AssertionError" message="expected:&lt;Failed&gt; but was:&lt;OK&gt;">
      <![CDATA[java.lang.AssertionError: expected:<Failed> but was:<OK>
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
	at FrontendTesting.PageControllerTesting.CreatePage(PageControllerTesting.java:106)
=======
  <testcase name="CreatePage" time="0.035" classname="FrontendTesting.PageControllerTesting">
    <error type="java.lang.AssertionError" message="Failed expected:&lt;mai&gt; but was:&lt;OK&gt;">
      <![CDATA[java.lang.AssertionError: Failed expected:<mai> but was:<OK>
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
	at FrontendTesting.PageControllerTesting.CreatePage(PageControllerTesting.java:99)
=======
<testsuite hostname="Nabila-PC" name="FrontendTesting.PageControllerTesting" tests="2" failures="0" timestamp="14 May 2015 17:33:08 GMT" time="3.814" errors="1">
  <testcase name="CreatePage" time="3.799" classname="FrontendTesting.PageControllerTesting"/>
  <testcase name="CreatePage" time="0.015" classname="FrontendTesting.PageControllerTesting">
    <error type="java.lang.AssertionError" message="expected [OK] but found [Failed]">
      <![CDATA[java.lang.AssertionError: expected [OK] but found [Failed]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at FrontendTesting.PageControllerTesting.CreatePage(PageControllerTesting.java:75)
>>>>>>> b4bce9d46968d253f312331916dbea989824bfcd
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- CreatePage -->
<<<<<<< HEAD
  <testcase name="CreatePage" time="0.029" classname="FrontendTesting.PageControllerTesting"/>
  <testcase name="ShowPageTime" time="0.005" classname="FrontendTesting.PageControllerTesting">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 2 parameters but the method FrontendTesting.PageControllerTesting#ShowPageTime takes 4 and TestNG is unable in inject a suitable object">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method FrontendTesting.PageControllerTesting#ShowPageTime takes 4 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
=======
<<<<<<< HEAD
  <testcase name="CreatePage" time="0.031" classname="FrontendTesting.PageControllerTesting">
    <error type="java.lang.AssertionError" message="OK expected:&lt;mai&gt; but was:&lt;OK&gt;">
      <![CDATA[java.lang.AssertionError: OK expected:<mai> but was:<OK>
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
	at FrontendTesting.PageControllerTesting.CreatePage(PageControllerTesting.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
<<<<<<< HEAD
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
    </failure>
  </testcase> <!-- ShowPageTime -->
  <testcase name="CreateMyTimelinePost" time="0.031" classname="FrontendTesting.PageControllerTesting">
=======
]]>
    </error>
  </testcase> <!-- CreatePage -->
  <testcase name="CreateMyTimelinePost" time="0.021" classname="FrontendTesting.PageControllerTesting">
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
    <error type="java.lang.AssertionError" message="Ok expected:&lt;n&gt; but was:&lt;OK&gt;">
      <![CDATA[java.lang.AssertionError: Ok expected:<n> but was:<OK>
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
<<<<<<< HEAD
	at FrontendTesting.PageControllerTesting.CreateMyTimelinePost(PageControllerTesting.java:154)
=======
	at FrontendTesting.PageControllerTesting.CreateMyTimelinePost(PageControllerTesting.java:143)
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- CreateMyTimelinePost -->
<<<<<<< HEAD
=======
=======
>>>>>>> b4bce9d46968d253f312331916dbea989824bfcd
>>>>>>> ed6cdda22f362a9816aed254c00b74fc2be43805
</testsuite> <!-- FrontendTesting.PageControllerTesting -->
